uncontrast_studies/models/finding.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.db.models import CASCADE
+from simple_history.models import HistoricalRecords
+
+
+class UnConFinding(models.Model):
+    experiment = models.ForeignKey(
+        null=False, blank=False, to="uncontrast_studies.UnConExperiment", on_delete=CASCADE, related_name="findings"
+    )
+    outcome = models.CharField(null=False, blank=False, max_length=250)
```

Comment:
looks like a TextField (with no char limitation)

Reply:
I thought so too at first but the actual data they sent doesn't look like a free text field, more like a short-answer / drop-down options field. I can double check.